source/accessibility.html.md.erb

```diff
@@ -49,12 +49,12 @@ GDS is committed to making its website accessible, in accordance with the Public
 
 ## Compliance status
 
-This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
```

Comment:
Has this changed, or was it incorrect before?

Reply:
It was incorrect before. The last TDT update made it only partially compliant but I missed this part when I updated the statement.